keyboards/southpaw75/keymaps/default/keymap.c

```diff
@@ -0,0 +1,16 @@
+#include QMK_KEYBOARD_H
```

Comment:
Please add a license header here.

Reply:
Apologies for the noob questions, but do you have an example of the license header I should be adding to all of the files you mentioned? Thanks!

Reply:
Absolutely:
https://github.com/qmk/qmk_firmware/blob/2d8703f6cc61918aaefd3ce04a064a82ab238140/quantum/template/avr/config.h#L1-L16

Just fill in the missing spots.

Reply:
Awesome, I will go ahead and add that to all the files you asked for it on. Thanks!